calc/vxlan_resolver.go

```diff
@@ -208,7 +211,8 @@ func (c *VXLANResolver) OnHostConfigUpdate(update api.Update) (_ bool) {
 	case "IPv4VXLANTunnelAddr":
 		nodeName := update.Key.(model.HostConfigKey).Hostname
 		vtepSent := c.vtepSent(nodeName)
-		logCxt := logrus.WithField("node", nodeName)
+		logCxt := logrus.WithField("node", nodeName).WithField("value", update.Value)
```

Comment:
I think it's a bit weird to add the "value" to the context logger, since we augment with the old / new IP below so we'll get duplicate info in the log I think?

Maybe just log it out in the debug line itself?